### ISSUE TITLE
On Windows 11, NVDA does not read Dbeaver autocomplete suggestions

### ISSUE THREAD

**Issue by thgcode** [nvaccess/nvda#16416]
### Steps to reproduce:
1. Open dbeaver, you can use the provided sample database.

2. Press CTRL+ALT+ENTER on the sample database to open the SQL console.

3. Start to type selec to autocomplete the select statement.

### Actual behavior:

You will notice NVDA speaks nothing.
But, if you use object navigation, you will find the autocomplete suggestions.
On a machine with Windows 10, it works, but on this machine with Windows 11 not.

### Expected behavior:

NVDA speaking the autocomplete suggestion for select.

#### NVDA installed/portable/running from source:
Installed / Running from source

#### NVDA version:
2023.1, 2024.1, source-master-11a2b4f

#### Windows version:
Windows version: Windows 11 23H2 (10.0.22631) workstation AMD64

#### Name and version of other software in use when reproducing the issue:
Dbeaver Version 24.0.2.202404071654

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
All versions tested don't speak the suggestions.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.
